core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -827,6 +831,14 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
   // visible for testing
   private[group] def offsetWithRecordMetadata(topicPartition: TopicPartition): Option[CommitRecordMetadataAndOffset] = offsets.get(topicPartition)
 
+  private[group] def pendingOffsetCommit(topicIdPartition: TopicIdPartition): Option[OffsetAndMetadata] = {
```

Comment:
Are those two methods for testing? If so, could we add a comment for each: `Used for testing`.

Reply:
Yes, they are used for testing exclusively.